enzyme/benchmarks/ReverseMode/ode/ode.cpp

```diff
@@ -25,6 +25,9 @@ static float tdiff(struct timeval *start, struct timeval *end) {
 void boost::throw_exception(std::exception const & e) {
     //do nothing
 }
+void boost::throw_exception(std::exception const & e, boost::source_location const & loc) {
```

Comment:
Is this needed? CI does not like it:

```cpp
Command Output (stderr):
--
ode.cpp:28:62: error: no type named 'source_location' in namespace 'boost'
void boost::throw_exception(std::exception const & e, boost::source_location const & loc) {
                                                      ~~~~~~~^
ode.cpp:28:78: error: expected ')'
void boost::throw_exception(std::exception const & e, boost::source_location const & loc) {
                                                                             ^
ode.cpp:28:28: note: to match this '('
void boost::throw_exception(std::exception const & e, boost::source_location const & loc) {
                           ^
2 errors generated.
```

Reply:
According to [here](https://www.boost.org/doc/libs/1_81_0/libs/throw_exception/doc/html/throw_exception.html) and [here](https://www.boost.org/doc/libs/1_81_0/libs/assert/doc/html/assert.html#source_location_support),  the overload of `throw_exception` that takes a `boost::source_location` object is required since Boost 1.73.0, but `boost::source_location` is only available after Boost 1.73.0. Maybe this is the problem, and I will add a version check.